WeakAuras/WeakAuras.lua

```diff
@@ -3613,6 +3613,73 @@ do
   LGF.RegisterCallback("WeakAuras", "FRAME_UNIT_REMOVED", frame_monitor_callback)
 end
 
+local anchor_action_bar_monitor
+do
+  local function actionBarFrameMonitorCallback(self, event, slotId)
```

Comment:
This is modeled a bit after the `frame_monitor_callback` above. I didn't use `frame_monitor_callback` directly because it looked to be geared for callbacks from `LGF`.